encord/client.py

```diff
@@ -165,6 +166,10 @@ class EncordClient:
     def __init__(self, querier: Querier, config: Config, api_client: Optional[ApiClient] = None):
         self._querier = querier
         self._config = config
+
+        if api_client is None:
+            api_client = ApiClient(config)
```

Comment:
I would encapsulate this into the `_get_api_client`, to replace the `ApiClient should exist when authenticated with SSH key` error